src/mame/machine/nb1412m2.cpp

```diff
@@ -179,7 +178,8 @@ void nb1412m2_device::device_reset()
 	m_rom_op = 0;
 	m_const90 = 0x18; // fixes coin sample if inserted at first title screen
 	m_dac_current_address = m_dac_start_address = 0;
-	m_dac_frequency = 4000;
+	m_timer_rate = 2;
+	m_dac_frequency = m_timer_rate * 71 * 18;
```

Comment:
71 times 18 is 1,278 as opposed to 1,280 in `dac_timer_w` – where did the numbers come from anyway?

Reply:
Oops, that was a careless mistake, I forgot to push the fixed source.
It's fixed.